src/NUnitFramework/framework/Constraints/Operators/ExactCountOperator.cs

```diff
@@ -29,7 +29,7 @@ namespace NUnit.Framework.Constraints
     /// Represents a constraint that succeeds if the specified 
     /// count of members of a collection match a base constraint.
     /// </summary>
-    public class ExactCountOperator : CollectionOperator
+    public class ExactCountOperator : SelfResolvingOperator
     {
         private int expectedCount;
```

Comment:
@rprouse I was going to refactor this as well since I made significant changes to the file, but then I realized it would be inconsistent with the base members `left_precedence` and `right_precedence` which are referenced in the added code. Shall I just leave this file as-is?